### PR TITLE
chore!: rename QGB to BlobStream

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

closes https://github.com/celestiaorg/orchestrator-relayer/issues/512

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# stage 1 Build qgb binary
-FROM --platform=$BUILDPLATFORM docker.io/golang:1.21.3-alpine3.18 as builder
+# stage 1 Build blobstream binary
+FROM --platform=$BUILDPLATFORM docker.io/golang:1.21.2-alpine3.18 as builder
```
Comment (SweeXordious) [inline Dockerfile:2]:
```suggestion
FROM --platform=$BUILDPLATFORM docker.io/golang:1.21.3-alpine3.18 as builder
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # orchestrator-relayer
 
-Contains the implementation of the QGB orchestrator and relayer.
+Contains the implementation of the Blobstream orchestrator and relayer.
 
-The orchestrator is the software that signs the QGB attestations, and the relayer is the one that relays them to the target EVM chain.
+The orchestrator is the software that signs the Blobstream attestations, and the relayer is the one that relays them to the target EVM chain.
 
-For a high-level overview of how the QGB works, check [here](https://github.com/celestiaorg/quantum-gravity-bridge/tree/76efeca0be1a17d32ef633c0fdbd3c8f5e4cc53f#how-it-works) and [here](https://blog.celestia.org/celestiums/).
+For a high-level overview of how the Blobstream works, check [here](https://github.com/celestiaorg/quantum-gravity-bridge/tree/76efeca0be1a17d32ef633c0fdbd3c8f5e4cc53f#how-it-works) and [here](https://blog.celestia.org/celestiums/).
```
Comment (SweeXordious) [inline README.md:7]:
```suggestion
For a high-level overview of how Blobstream works, check [here](https://github.com/celestiaorg/quantum-gravity-bridge/tree/76efeca0be1a17d32ef633c0fdbd3c8f5e4cc53f#how-it-works) and [here](https://blog.celestia.org/celestiums/).
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
 # orchestrator-relayer
 
-Contains the implementation of the QGB orchestrator and relayer.
+Contains the implementation of the Blobstream orchestrator and relayer.
 
-The orchestrator is the software that signs the QGB attestations, and the relayer is the one that relays them to the target EVM chain.
+The orchestrator is the software that signs the Blobstream attestations, and the relayer is the one that relays them to the target EVM chain.
```
Comment (SweeXordious) [inline README.md:5]:
```suggestion
The orchestrator is the software that signs Blobstream attestations, and the relayer is the one that relays them to the target EVM chain.
```

**Comment by SweeXordious** [comment]
When you have time @evan-forbes,  can't ping you on Slack as it's down for me since yesterday and only works occasionaly

**Review by evan-forbes — APPROVED** [review]
lgtm. I will catch up on reviews today, the past few days have been rather constrained on time

**Pull request merged**